Add missing include for POSIX basename, since systemd-logind expects GNU basename in string.h

diff -rup src/basic/musl_missing.h.orig src/basic/musl_missing.h
--- a/src/basic/musl_missing.h
+++ b/src/basic/musl_missing.h
@@ -25,6 +25,7 @@ void elogind_set_program_name(const char
 #include <string.h>
 #include <unistd.h>
 #include <pthread.h> /* for pthread_atfork */
+#include <libgen.h> /* for POSIX basename */
 
 /*
  * Possibly TODO according to http://man7.org/linux/man-pages/man3/getenv.3.html
